### PR TITLE
Add git-paste for sending patches to pastebin

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

**Comment by pabs3** [comment]
I've fixed the test failures with this PR.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

File: bin/git-paste [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
```
Comment (spacewander) [inline bin/git-paste]: Better to add `-o pipefail` to ensure the script exits with non-zero code when `git format-patch` failed.

**Comment by pabs3** [comment]
Fixed the examples (copy-paste issues) and added pipefail.

Selection of the target pastebin should be done by updating the
pastebinit configuration. The documentation for git paste mentions
pastebinit so I assume people will be able to figure that out, do you
think that should be explicitly mentioned?

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by spacewander** [comment]
@pabs3 
Maybe we could add a link to the online pastebinit configuration documentation?

**Comment by pabs3** [comment]
The only documentation I can find for pastebinit is the manual page and
the Ubuntu help wiki page. The wiki doesn't indicate how to configure
pastebinit and the manual page doesn't have a canonical online location
but there are online versions from Debian, Ubuntu and Hurricane
Electric (HE's copy is unencrypted). 

https://help.ubuntu.com/community/Pastebinit
https://manpages.debian.org/pastebinit
https://manpages.ubuntu.com/cgi-bin/search.py?q=pastebinit
http://man.he.net/man1/pastebinit

Which URL should I use for the documentation?

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by spacewander** [comment]
@pabs3 
Personally speaking, I vote for the https://manpages.debian.org/pastebinit

**Comment by pabs3** [comment]
I've added a configuration section to the manual page and a sentence
about pastebinit configuration to the Commands.md file.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by spacewander** [comment]
@pabs3 
Merged. Thanks!

**Comment by pabs3** [comment]
Awesome, thanks!

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Pull request merged**